staging/src/k8s.io/kube-aggregator/pkg/apiserver/openapi_aggregator.go

```diff
@@ -132,6 +166,9 @@ func sortByPriority(specs []openAPISpecInfo) {
 		groupPriorities: map[string]int32{},
 	}
 	for _, spec := range specs {
+		if spec.apiService.Spec.Service == nil {
```

Comment:
we skip local services in the order, but still add this special case in the comparison function?